### PR TITLE
hardening/117_postIndividualContextEntityAttributeWithTypeAndId

### PR DESCRIPTION
**Pull request by kzangeli**
One more (less) conv op:
POST /v1/contextEntities/type/{entity::type}/id/{entity::id}/attributes/{attribute::name}

### CODE REVIEW

File: test/functionalTest/cases/117_convop_using_standard_ops/postIndividualContextEntityAttributeWithTypeAndId.test [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0
+
+--SHELL--
+
+#
+# 01. POST /v1/contextEntities/type/T1/id/E1/attributes/A1, A1=1
+# 02. GET /v1/contextEntities/type/T1/id/E1/attributes/A1
+# 03. POST /v1/contextEntities/type/T1/id/E1/attributes/A1?entity::type=T2, and see it fail
```
Comment (fgalan) [inline test/functionalTest/cases/117_convop_using_standard_ops/postIndividualContextEntityAttributeWithTypeAndId.test]: What about "POST /v1/contextEntities/type/T1/id/E1/attributes/A1?entity::type=T1" and see it works?

(What about similar .tests in previous PR? Do they include this step?)

Comment (kzangeli) [inline test/functionalTest/cases/117_convop_using_standard_ops/postIndividualContextEntityAttributeWithTypeAndId.test]: I normally include that step ...
Might have forgotten it here

Comment (kzangeli) [inline test/functionalTest/cases/117_convop_using_standard_ops/postIndividualContextEntityAttributeWithTypeAndId.test]: Fixed in bef25b4

**Comment by fgalan** [comment]
LGTM

**Pull request merged**